backend/compact-connect/lambdas/python/provider-data-v1/tests/function/test_handlers/test_provider_users.py

```
@@ -384,6 +384,33 @@ def test_patch_provider_military_affiliation_updates_status(self):
         self.assertEqual(1, len(affiliation_record))
         self.assertEqual('inactive', affiliation_record[0]['status'])
 
+    def test_patch_provider_military_affiliation_updates_status_when_initializing(self):
+        from handlers.provider_users import provider_user_me_military_affiliation
```

Pylint (3.3.7)

[convention] Import outside toplevel (handlers.provider_users.provider_user_me_military_affiliation)

(C0415)